Library/DuetBdsLib/BdsPlatform.c

```diff
@@ -1091,14 +974,6 @@ Routine Description:
     DetectAndPreparePlatformPciDevicePath (TRUE);
   }
 
-  //
-  // The ConIn devices connection will start the USB bus, should disable all
-  // Usb legacy support firstly.
-  // Caution: Must ensure the PCI bus driver has been started. Since the
-  // ConnectRootBridge() will create all the PciIo protocol, it's safe here now
-  //
-  DisableUsbLegacySupport ();
```

Comment:
Is it really not needed?

Reply:
Removed in favor of the PCDs checked by UhciDxe and EhciDxe. This prevents legacy USB from being turned off on the non-USB DUET (BlockIo).